### ISSUE TITLE
coding: develop the tech talk section in events page

### ISSUE THREAD

**Issue by nandini45** [anitab-org/anitab-org.github.io#122]
## Description
As a developer, I need to see all the videos of the talks held by the community regarding open source and anitab-org , so that the user can know more about us and what we do.

## Mocks
![image](https://user-images.githubusercontent.com/62046625/89104902-4e980a00-d43a-11ea-98cf-78835c31c020.png)

the spacing is 16 from both sides

### Update [Required]
- [ ]  Follows all the design guidelines as mentioned and is responsive enough on large screen devices.
- [ ]  the talk should be as mentioned in the doc

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 2 mentor.

## Estimation
3 hours

**Comment by imrishabh18** [comment]
Can I take up this issue @nandini45? If it's not in hold now

**Comment by nandini45** [comment]
@nandini45  ah ok
actually the video url has to be mentioned in the section 
when u come to that part tell me i will provide you with all the links.

**Comment by imrishabh18** [comment]
@nandini45 Can I install a carousel package in react for this feature?

**Comment by nandini45** [comment]
yeah sure @imrishabh18

**Comment by nandini45** [comment]
any updates?

**Comment by imrishabh18** [comment]
Yeah, I have made the carousel and the structure. Just the styling part is left. Will push the update today.

**Comment by Bucky25** [comment]
if nobody is working can i work on this  @nandini45

**Comment by Bucky25** [comment]
@nandini45  cam i use react-bootstrap in it or not

**Comment by nandini45** [comment]
@Bucky25 i think the person working on it has created a PR already you can work on other issue..i think recently 2 new issues were made available..you can check them

**Comment by nandini45** [comment]
#123 and #80 one

**Comment by tichnas** [comment]
@imrishabh18 Are you working on the changes requested in the PR?
If not, will it be ok if I work on this issue?
Thanks

**Comment by tichnas** [comment]
@nandini45 Can I please be assigned to this issue?

Also, I have a doubt that the section where the VIDEO is written is actually a video or an image which when clicked will open a link containing the video (looking at the closed PR of this issue it seems it's just an image, that too without any link)?

Thanks

**Comment by nandini45** [comment]
@tichnas video have to be there. I will provide the url to them and you can use it.
I assigned it to u..

**Comment by tichnas** [comment]
@nandini45 Unfortunately, I'm kind of at a dead-end at implementing the video.

For displaying a video in react-native, we use a package called `react-native-video`, but it only works on android and ios.
I found the only possible solution to make it work with `react-native-web` is dynamically creating a `video` element using `react-native-web`. In short, then it will only work on the web, and we will have to do some changes later (when we create an app) to make it work on android and ios also (refer https://stackoverflow.com/a/59333885/13604425 and https://github.com/necolas/react-native-web/issues/1385).

Now, can you please confirm if I should go with the solution I found or not?

**Comment by nandini45** [comment]
@annabauza  can you please help us here??

**Comment by chinmaym07** [comment]
Hey @nandini45  Can I Try work on this issue .?

**Comment by nandini45** [comment]
@chinmaym07 , @tichnas he is already working on it. 
You can look for other issues probably

**Comment by shruti1421** [comment]
If no one is working on this issue, can I try?

**Comment by nandini45** [comment]
Sure smiriti

**Comment by nandini45** [comment]
@shruti1421 i think i assigned you one issue already.
after probably making the PR of that issue i can assign you this issue

**Comment by HarshKumarChoudary** [comment]
Since I am a bit familiar with javascript and css and If this issue is available I would like to work on it.
